include/SampleTrack.h

```diff
@@ -73,6 +73,15 @@ class SampleTCO : public TrackContentObject
 	bool isPlaying() const;
 	void setIsPlaying(bool isPlaying);
 
+	void inline setMarkerPos( int x )
+	{
+		m_markerPos = x;
+	}
+	void inline setMarkerEnabled( bool e )
+	{
+		m_marker = e;
+	}
+
```

Comment:
Let's remove the spaces inside the brackets in new code.

Reply:
I thought policy was to match surrounding code until an autoformatter fixes this stuff.

Reply:
> match surrounding code

`setIsPlaying()` doesn't have spaces in the parentheses, but `setSampleBuffer()` does. Neither way will match the surrounding code perfectly.

Reply:
The vast majority of the file uses spaces, but fair point.